ui/src/app/cohort-review/routing/routing.module.ts

```diff
@@ -27,28 +28,19 @@ const routes: Routes = [{
       concepts: DemographicConceptMapsResolver,
     },
     data: {
-      breadcrumb: {
-        value: 'Participants',
-        intermediate: true
-      },
+      breadcrumb: 'cohort'
     }
   }, {
     path: 'participants/:pid',
     component: DetailPage,
     data: {
-      breadcrumb: {
-        value: 'Participant :pid',
-        intermediate: false
-      }
+      breadcrumb: 'participant'
     }
   }, {
     path: 'report',
     component: QueryReportComponent,
     data: {
-      breadcrumb: {
-        value: 'Query Report',
-        intermediate: false
-      }
+      breadcrumb: 'report'
```

Comment:
This doesn't appear in `getTrail()` (I'm assuming it should). An enum rather than a string might avoid these desynchronizing and make the linkage clearer.

Reply:
Good catch, although it turns out this page isn't actually reachable any more, as its content is now shown as the second tab on the participants page. I removed the route entry instead.